ui/src/i18n/de/translation.json

```diff
@@ -0,0 +1,25 @@
+{
+  "common": {
+    "loading": "Laden...",
```

Comment:
why not Polish but German? xD

Reply:
The ticket didn't specify what language, so I decided to go with German :)